notification-settings/lib/notification_settings/notification_library.rb

```diff
@@ -44,14 +44,15 @@ def status_allows_create?
       end
 
       def settings_allow_create?
-        target.notification_setting.settings.dig(:enabled)
+        target.notification_setting.settings.fetch(:enabled) { true }
       end
 
       def category_settings_allow_create?
-        target.notification_setting.category_settings.dig(
+        value = target.notification_setting.category_settings.dig(
           category.to_sym,
           :enabled
         )
+        value.nil? : true : value
```

Comment:
Isn't this supposed to be?

```suggestion
        value.nil? ? true : value
```